<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>threshold1_slider</class>
 <widget class="QMainWindow" name="threshold1_slider">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SOA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>961</width>
      <height>791</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>30</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Upload</string>
     </attribute>
     <widget class="QToolButton" name="uploadFile">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>240</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QFrame" name="previewPhoto">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>911</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="UploadFileName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>631</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="NextButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>630</y>
        <width>144</width>
        <height>41</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>881</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Mongolian Baiti</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string>SOA</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>130</y>
        <width>841</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>morphological parameters</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Automatic identification of dendritic trees' </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Properties</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>411</width>
        <height>141</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Edges</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>141</width>
         <height>33</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Threshold :</string>
       </property>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>40</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSpinBox" name="threshold1_slider_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>511</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Merge Lines</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>39</y>
         <width>311</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Min distance to merge :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>89</y>
         <width>321</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Min angle to merge :</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="min_distance">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>40</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="min_angel">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>90</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>30</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_ok">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>630</y>
        <width>144</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QFrame" name="threshold1_slider_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>941</width>
        <height>371</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_create_preview">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>481</width>
         <height>51</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Create Preview Segmentation Image</string>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
